tools/parse-release-notes

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env node
```

Comment:
Almost the same as for integreat. Just removes web-option and language-selection, as lunes is just available in german.